src/components/portrait-view.tsx

```diff
@@ -116,43 +132,58 @@ export const PortraitView = observer(function PortraitView(props: ITableProps) {
     const precisions = getAttrPrecisions(collections);
     const attrTypes = getAttrTypes(collections);
     const attrVisibilities = getAttrVisibility(collections);
+    const focusSetForLevels = new Map<number, boolean>();
+
+    const updateFocusSetForLevel = (level: number) => {
+      focusSetForLevels.set(level, true);
+    };
```

Comment:
Can you explain the purpose of this focus set code?

From what I can tell:
- on each render of the portrait view table the focusSetForLevels is reset to a new map
- when updateFocusSetForLevel is called it "sets the focus" for that level.
- when the PortraitViewRow is rendered it "sets the focus" for its parent level if it isn't already set.
- the `shouldGetFocusOnNewAttribute` is based on whether the parent of the row focus

Is the idea to try to track whether the a editable header is being edited (so it has focus)?

Reply:
@scytacki The idea is to determine which instance of a new attribute's column header should get focus when a new attribute is added.

There can be multiple tables for a single level in the hierarchy. In that situation, when a new attribute is added to that level, only the first table's column header for that attribute should become editable. If we don't track that with the `focusSetForLevels` map and `shouldGetFocusOnNewAttribute`, the last/bottom table's column header for the new attribute will get focus instead.

I know this isn't ideal. If you have any thoughts on how it might be better handled given that added context, please let me know. If not, I can at least add a comment explaining the purpose.

Reply:
This is what I see when I try it in CODAPv3:
<img width="765" alt="image" src="https://github.com/user-attachments/assets/85f18308-dd56-45cb-8cf7-09eea7aaa9b9">

If I understand your intention right, this doesn't seem to be working. I think the reason is because the code assumes the nested tables will be rendered from top to bottom so the first one rendered will get the `shouldGetFocusOnNewAttribute`. I'm pretty sure the order of the rendering will be random. So this approach won't work. I can't think of a solution off the top of my head, but maybe if we pair we can come up with something. Or maybe this isn't really that important to get right.